website/docs/core-concepts/projects/configuration/stores.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Configure Stores
+sidebar_position: 4
+sidebar_label: Configure Stores
+id: stores
+---
+import Intro from '@site/src/components/Intro'
+
+<Intro>
+Atmos supports the concept of remote stores to facilitate the sharing of values between components or between
+some external process and a component. In `atmos`, values are saved to stores via
+[hooks](/core-concepts/stacks/hooks) and are read using the [!store](/core-concepts/stacks/yaml-functions/store.mdx)
+yaml function. Values can also be saved to stores from outside of `atmos`, for example, from a CI/CD pipeline or a
+script.
+
+Currently, the following stores are supported:
+
+- [AWS SSM Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html)
+- [Artifactory](https://jfrog.com/artifactory/)
+</Intro>
+
+Atmos stores are configured in the `atmos.yaml` file and available to use in stacks via the
+[store](/core-concepts/stacks/yaml-functions/store) YAML function.
+
+## CLI Configuration
+
+All of these settings should be configured in the [Atmos CLI Configuration](/cli/configuration) found in `atmos.yaml`.
+
+### Artifactory
+
+```yaml
+stores:
+  dev/artifactory:
+    type: artifactory
+    options:
+      url: https://cptrialstores.jfrog.io/artifactory
+      repo_name: tfsharedstore
+```
+
+<dl>
+  <dt>`stores.[store_name]`</dt>
+  <dd>This map key is the name of the store. It must be unique across all stores. This is how the store is referenced in the `store` function.</dd>
+
+  <dt>`stores.[store_name].type`</dt>
+  <dd>Must be set to `artifactory`</dd>
+
+  <dt>`stores.[store_name].options`</dt>
+  <dd>A map of options specific to the store type. For Artifactory, the following options are supported:</dd>
+
+  <dt>`stores.[store_name].options.access_token (optional)`</dt>
+  <dd>An access token to use for authentication. This is not recommended as it is less secure than using the
+  `JFROG_ACCESS_TOKEN` or `ARTIFACTORY_ACCESS_TOKEN` environment variables. See [Authentication](#authentication) below
+  for more information.</dd>
+
+  <dt>`stores.[store_name].options.prefix (optional)`</dt>
+  <dd>A prefix path that will be added to all keys stored or retreived from SSM Parameter Store. For example if the prefix
+  is `/atmos/infra-live/`, and if the stack is `plat-us2-dev`, the component is `vpc`, and the key is `vpc_id`, the full path
+  would be `/atmos/infra-live/plat-us2-dev/vpc/vpc_id`.</dd>
+
+  <dt>`stores.[store_name].options.repo_name (required)`</dt>
+  <dd>The name of the Artifactory repository to use.</dd>
+
+  <dt>`stores.[store_name].options.url (required)`</dt>
+  <dd>The URL of the Artifactory instance.</dd>
+</dl>
+
+#### Authentication
+
+The Artifactory store supports using an access token for authentication. The access token can be set directly in the
+`atmos.yaml` (not recommended) or via the `JFROG_ACCESS_TOKEN` or `ARTIFACTORY_ACCESS_TOKEN` environment variables.
```

Comment:
We were going to update the language here to point out that is not recommended to hard code static credentials, but you can use the ENV function.

Reply:
Yes, still have a couple more changes to submit (this included). I should have opened as a draft.